src/functional/geometry/convex_hull.zig

```diff
@@ -204,3 +345,77 @@ test "incremental convex hull random elements" {
         try testing.expectEqual(turn, Turn.left);
     }
 }
+
+test "Compute MABR LinearFunction for known ConvexHull one" {
+    const allocator = std.testing.allocator;
+
+    // Define a set of points forming a simple rectangle.
+    const points = [_]DiscretePoint{
+        .{ .time = 0, .value = 0.0 },
+        .{ .time = 1, .value = 3.0 },
+        .{ .time = 2, .value = 2.0 },
+        .{ .time = 3, .value = 5.0 },
+        .{ .time = 4, .value = 4.0 },
+    };
+
+    // Initialize the convex hull and add points to it.
+    var convex_hull = try ConvexHull.init(allocator);
+    defer convex_hull.deinit();
+
+    for (points) |point| {
+        try convex_hull.addPoint(point);
+    }
+
+    // Calculate the minimum-area bounding rectangle (MABR).
+    const mabr_linear_function = try convex_hull.computeMABRLinearFunction();
+
+    // Known rectangle area and known LinearFunction y=x+1
+    try testing.expect(@abs(mabr_linear_function.slope - 1) <= 0.0001);
+    try testing.expect(@abs(mabr_linear_function.intercept - 1) <= 0.0001);
+}
+
+test "Compute MABR LinearFunction for known ConvexHull two" {
+    const allocator = std.testing.allocator;
+
+    // Define a set of points forming a simple rectangle.
+    const points = [_]DiscretePoint{
+        .{ .time = 0, .value = 0.0 },
+        .{ .time = 1, .value = -2.0 },
+        .{ .time = 2, .value = 3.0 },
+        .{ .time = 3, .value = -1.0 },
+        .{ .time = 4, .value = 2.0 },
+    };
+    // Initialize the convex hull and add points to it.
+    var convex_hull = try ConvexHull.init(allocator);
+    defer convex_hull.deinit();
+
+    for (points) |point| {
+        try convex_hull.addPoint(point);
+    }
+
+    // Calculate the minimum-area bounding rectangle (MABR).
+    const mabr_linear_function = try convex_hull.computeMABRLinearFunction();
+
+    // Known rectangle area and known LinearFunction y=x+1
```

Comment:
Seems only the function is returned and not the MABR, maybe update the comment to reflect this.

Reply:
fixed.